.github/MAPS_AND_AWAY_MISSIONS.md

```diff
@@ -0,0 +1,27 @@
+## MAPS
+
+/tg/station currently comes equipped with five maps.
+
+* [BoxStation (default)](https://tgstation13.org/wiki/Boxstation)
+* [MetaStation](https://tgstation13.org/wiki/MetaStation)
+* [DeltaStation](https://tgstation13.org/wiki/DeltaStation)
+* [PubbyStation](https://tgstation13.org/wiki/PubbyStation)
+* [DonutStation](https://tgstation13.org/wiki/Donutstation)
```

Comment:
What about Kilo? It's not enabled on the servers but it still exists in the codebase.

Reply:
do not care that much tbh